docs/source/pkg_connector.rst

```diff
@@ -0,0 +1,28 @@
+PKG Connector
+=============
+
+This component is responsible for establishing a connection to the PKG and executing SPARQL queries against it.
+The SPARQL queries are generated based on the user's annotated statements using utility functions.
```

Comment:
Can we link the file with utility function here?

Reply:
Done.